app/controllers/schools/timeline_controller.rb

```diff
@@ -5,11 +5,11 @@ def show
       @academic_year = if params[:academic_year]
                          AcademicYear.find(params[:academic_year])
                        else
-                         @school.calendar_area.academic_year_for(Time.zone.today)
+                         @school.academic_year_for(Time.zone.today)
                        end
       first_observation = @school.observations.order('at ASC').first
       @active_academic_years = if first_observation
-                                 @school.calendar_area.parent_area.academic_years.where('end_date >= ? AND start_date <= ?', first_observation.at, Time.zone.today)
+                                 @school.calendar.based_on.academic_years.where('end_date >= ? AND start_date <= ?', first_observation.at, Time.zone.today)
```

Comment:
Are academic years at regional level now? Or at national level? This look like it would be doing the lookup on the regional level but the old implementation did another hop up to the calendar's area's parent area.

Reply:
Ah yes, I see what you mean - they are at the national level I think, so, it *should* either be @school.template_calendar.based_on.academic_years or @school.calendar.based_on.based_on.academic_years...

Reply:
I think I was getting confused myself!